### ISSUE TITLE
Labelling Training Set - Bounding Boxes

### ISSUE THREAD

**Issue by optimaltaskin** [open-mmlab/mmdetection#7823]
Hi,

I have a question about training set labelling. Because I'm trying to detect an objects mask, I have been labelling my images as below:

![Screenshot from 2022-04-25 08-29-55](https://user-images.githubusercontent.com/85254524/165026256-fdf98d5e-cf88-45aa-8fe3-d321f5946bd2.png)

I only selected mask boundaries for mask detection and didn't select any bounding boxes. Do I need to select bounding boxes as well for more proper results? Moreover when I don't select bounding boxes, how the algorithm compares its results with the test set since there is no bounding box in the set?

**Comment by Czm369** [comment]
Maybe you can generate bboxes from masks, referring to 
https://github.com/open-mmlab/mmdetection/blob/c72bc707e661d61cf09aca0a53ad21812ef874d0/mmdet/core/mask/utils.py#L68

**Comment by optimaltaskin** [comment]
@Czm369 thanks for your response. However I wonder if it is required to draw bounding boxes as well? I mean does the result of training change when I use a set that only contains mask labels and another set that contains both mask and bounding box labels?
